packages/Statie/packages/FlatWhite/src/Latte/LatteRenderer.php

```diff
@@ -15,8 +15,14 @@
 
     /**
      * @var string
+     * @see https://regex101.com/r/9jglKd/2/
      */
-    private const MATCH_PLACEHOLDERS = '#^' . self::PLACEHOLDER_PREFIX . '[0-9]+$#m';
```

Comment:
Diff is not really helpful. I changed it here to allow inline placeholders, like:

"Some text __placeholder and more text."

### Before

Only `__placeholder` on standalone line was allowed.

```php
#^' . self::PLACEHOLDER_PREFIX . '[0-9]+$#m';
```

### After

```php
#' . self::PLACEHOLDER_PREFIX . '[0-9]+#m';
```